src/components/statusBox/index.jsx

```diff
@@ -0,0 +1,34 @@
+export default function StatusBox({ invoiceStatus, invoiceDue }) {
+    let dateString = `${invoiceDue}T00:00:00`
+    let dateDue = Date.parse(dateString)
+    const todaysDate = new Date()
+    let day = todaysDate.getDate()
+    const year = todaysDate.getFullYear()
+    let month = todaysDate.getMonth()
+    month = month + 1
+    if (day < 10) {
+        day = `0${day}`
+    }
+    if (month < 10) {
+        month = `0${month}`
+    }
+    const todaysDateFormat = `${year}-${month}-${day}T00:00:00`
+    let todayDateParsed = Date.parse(todaysDateFormat)
```

Comment:
Really this should all go into a useEffect, every time state in this component, or it's parents, changes, all of this code will re-run. It won't cause any issues now, because state only changes when the fetch request completes, but when you start doing future stories it will.

Move it into a useEffect, and then make some state to contain the results you're actually using in your JSX (todayDateParsed and dateDue)